includes/Story_Post_Type.php

```diff
@@ -552,6 +571,11 @@ public function get_editor_settings() {
 				'metadata'              => [
 					'publisher' => $this->get_publisher_data(),
 				],
+				'postLock'              => [
+					'interval'         => $time_window,
+					'showLockedDialog' => $show_locked_dialog,
+					'api'              => rest_url( sprintf( '/web-stories/v1/%s/%s/lock', $rest_base, $story_id ) ),
```

Comment:
A bit odd that this isn't under `api` above like all the other endpoints. Let's move it there, like we also did for meta boxes.